src/test/analyzer_test.ts

```diff
@@ -12,15 +12,17 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-'use strict';
+/// <reference path="../../node_modules/@types/mocha/index.d.ts" />
 
-const assert = require('chai').assert;
-const path = require('path');
-const BuildAnalyzer = require('../lib/analyzer').BuildAnalyzer;
-const waitForAll = require('../lib/streams').waitForAll;
-const sinon = require('sinon');
-const ProjectConfig = require('polymer-project-config').ProjectConfig;
-const Writable = require('stream').Writable;
+import {assert} from 'chai';
+import * as path from 'path';
+import File = require('vinyl');
```

Comment:
We already do :/ But because `vinyl` exports a class and not a true module (at least according to it's typedef) we can't use the `import ... from ...` syntax and need to continue using `require()`.

This is common-ish so I don't think it needs a comment, just one of those weird parts of TS + ES syntax.